src/components/common/ExtensionsMetadata.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <button
+    :disabled="isBusy"
+    type="button"
+    @click="updateMetadata"
+    class="my-1 inline-flex justify-center w-full border border-gray-300 dark:border-darkGray-600 px-2 py-1 bg-white dark:bg-darkGray-900 text-xs font-light text-gray-700 dark:text-darkGray-100 hover:bg-gray-100 dark:hover:bg-darkGray-700 focus:outline-none focus:ring focus:ring-gray-100 dark:focus:ring-darkGray-600"
```

Comment:
![Screen Shot 2021-07-19 at 5 02 13 PM](https://user-images.githubusercontent.com/40356749/126124923-cc0bb0c8-bd33-4121-aad1-5b14053e505f.png)

the metadata update button looks to be square. Can you update the style so that it matches the ones on the page? https://github.com/PlasmNetwork/plasm-apps-vue/blob/58e0d36d6dfc9a7201260fc51b9055c29051df05/src/components/balance/ModalTransferAmount.vue#L76